beets/library.py

```diff
@@ -66,10 +66,6 @@ def __init__(self, field, pattern, fast=True, case_sensitive=None):
             case_sensitive = platform.system() != 'Windows'
         self.case_sensitive = case_sensitive
 
-        # Use a normalized-case pattern for case-insensitive matches.
-        if not case_sensitive:
-            pattern = pattern.lower()
-
```

Comment:
It occurs to me that, if we remove this, we'll also need to change the `match` method below (which currently assumes both `self.file_path` and `self.dir_path` are lower-cased. Maybe we should just leave this in.